lib/eex/test/eex/tokenizer_test.exs

```diff
@@ -374,8 +374,16 @@ defmodule EEx.TokenizerTest do
   end
 
   test "returns error when there is start mark and no end mark" do
+    message = """
+    missing token '%>'
+
+      |
+    1 | foo <% :bar
+      |        ^
+    """
```

Comment:
I think in this case we want to point to the end? Or we want to adjust the test to something like:

```suggestion
    message = """
    expected closing '%>' for EEx expression

      |
    1 | foo <% :bar
      |        ^
    """
```